api/src/main/java/org/openmrs/module/appframework/service/AppFrameworkServiceImpl.java

```diff
@@ -421,5 +422,24 @@ public void purgeUserApp(UserApp userApp) {
         new AppFrameworkActivator().contextRefreshed();
     }
 
+	@Override
+	public List<Extension> getAllExtensions() {
+
```

Comment:
@HerbertYiga , is there no already existing method that can do the work this introduced method is doing?

Reply:
@Ruhanga Not really thats why i had to introduce it within the AppframeworkService  cc @dkayiwa

Reply:
So you are sure [this method](https://github.com/openmrs/openmrs-module-appframework/blob/master/api/src/main/java/org/openmrs/module/appframework/service/AppFrameworkServiceImpl.java#L185) doesn't do the job.

Reply:
@Ruhanga when i look at that  method,it only gets the enabled extensions,and what happens t the extensions that are not yet enabled!! thats why i had to call up all the extensions for the enabled Apps. cc @dkayiwa